File: Uncensorapi/uncensorapi.swift
import Foundation

let url = URL(string: "https://goodbyesyntax-uncensorapi.glitch.me/uncensor")!
var request = URLRequest(url: url)
request.httpMethod = "POST"
request.setValue("application/json", forHTTPHeaderField: "Content-Type")

let body = ["text": "Th1s 1s @ test"]
request.httpBody = try? JSONSerialization.data(withJSONObject: body)

let task = URLSession.shared.dataTask(with: request) { data, _, _ in
    if let data = data,
       let json = try? JSONSerialization.jsonObject(with: data) as? [String: Any],
       let uncensored = json["uncensored"] as? String {
        print(uncensored)
    }
}
task.resume()

File: Uncensorapi/uncensorapi.java
import java.io.*;
import java.net.*;
import org.json.JSONObject;

public class Uncensor {
    public static void main(String[] args) throws Exception {
        String input = "{\"text\": \"Th1s 1s @ test\"}";
        URL url = new URL("https://goodbyesyntax-uncensorapi.glitch.me/uncensor");
        HttpURLConnection con = (HttpURLConnection) url.openConnection();

        con.setRequestMethod("POST");
        con.setRequestProperty("Content-Type", "application/json; utf-8");
        con.setDoOutput(true);

        try(OutputStream os = con.getOutputStream()) {
            byte[] inputBytes = input.getBytes("utf-8");
            os.write(inputBytes, 0, inputBytes.length);
        }

        BufferedReader br = new BufferedReader(new InputStreamReader(con.getInputStream(), "utf-8"));
        StringBuilder response = new StringBuilder();
        String responseLine;
        while ((responseLine = br.readLine()) != null) {
            response.append(responseLine.trim());
        }

        JSONObject json = new JSONObject(response.toString());
        System.out.println(json.getString("uncensored"));
    }
}
